cms/static/js/spec/views/settings/main_spec.js

```diff
@@ -72,12 +72,12 @@ define([
             );
         });
 
-        it('Changing course start date without active certificate configuration should result in error', function () {
-            this.view.$el.find('#course-start-date')
+        it('Changing course start date without active certificate configuration should not result in error',
```

Comment:
I'm okay leaving the test in place -- it may catch invalid behavior at some point going forward.